### DISCUSSION TITLE
nav:collection:pages :from="segment_1" - Not working without {parent_uri}

### DISCUSSION THREAD

**Discussion by niloc7** [statamic/cms#4812] [Troubleshooting]
Perhaps this is a bug.. or a missing feature?

Yes, obviously it needs a {parent_uri} in order to use segment_1 or uri.

But what do you do to get the child pages as links for a particular page, if you don't want the links to be in the format of "{parent_uri}/{slug} and simply just the {slug} instead?

**Comment by jackmcdade** [comment]
Use {{ url }} instead :)

**Reply by niloc7** [reply]
Oh no, I think I must have explained wrongly..

I'm wondering how to load the child pages as a nav bar, WITHOUT using a route of {parent_uri}/{slug}, but just {slug} instead.

Cause by doing that, then {{ nav:collection:pages :from="segment_1"}} will not work when in the child pages.

**Comment by jackmcdade** [comment]
Okay, I guess I don't understand what you're asking. Can you share an example of what HTML you'd like with actual urls?

**Reply by niloc7** [reply]
Assuming we are in www.coke.com

Within that site, I have 10 flavours. So www.coke.com/raspberry, etc.

So within each flavor, I want an Introduction, Product Info, Testimonials, and Resources.

I made it so that all those pages are child of the Raspberry main page.

So I have www.coke.com/raspberry/introduction, etc.

In all those pages, I have a SIDE NAV like this:

Introduction (which links to the Raspberry main page)
Product Info
Testimonials
Resources

All works well if we do this:

```
{{ nav:collection:types :from="segment_1"}}
<a href="{{ url }}">{{ title }}</a>
{{ /nav:collection:types }}
```

However, once I configure the collection Route, changing from this "{parent_uri}/{slug}" to this {slug}" then it doesn't work anymore.

Is there something other than "segment_1" that you can use to denote all child links from their shared parent?

---

In case you're wondering why I need to do the routing this way, this is cause I need to name all the slugs as raspberry-introduction, raspberry-resources, etc.

Cause if I don't, then for the other flavors, the introduction will become introduction.2.md, introduction.3.md, for each parent flavor (raspberry, banana, mango, etc).

So I thought the solution would be by giving the slugs more complete names.. but in doing so, then I face this new problem.

Sorry for the lengthy problem =(
